### PR TITLE
Add 'source' object code for 'stop_point'

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: src/gtfs/read.rs [hunk 1]
```diff
@@ -83,14 +83,15 @@ impl From<Agency> for objects::Company {
 
 impl From<Stop> for objects::StopArea {
     fn from(stop: Stop) -> objects::StopArea {
-        let mut stop_codes: KeysValues = BTreeSet::new();
+        let mut codes = KeysValues::default();
+        codes.insert(("source".to_string(), stop.id.clone()));
         if let Some(c) = stop.code {
-            stop_codes.insert(("gtfs_stop_code".to_string(), c));
+            codes.insert(("gtfs_stop_code".to_string(), c));
```
Comment (datanel) [inline src/gtfs/read.rs]: Where does `gtfs_stop_code` come from? I don't see it in the specs

Comment (woshilapin) [reply src/gtfs/read.rs]: This was already here before. I discussed with @papailio and she seemed aware of that `gtfs_stop_code` being added in the `object_code`. Maybe we should just update the specifications (I can do it as part of this PR).
@prhod @papailio

**Comment by woshilapin** [comment]
I fixed the tests for this PR in 157f54c16aadac157488371b0013c49cb66c2eba.

File: src/gtfs/read.rs (hunk 1 above)
Comment (ArnaudOggy) [reply src/gtfs/read.rs]: Specs updated in https://github.com/CanalTP/transit_model/pull/403

Comment (woshilapin) [reply src/gtfs/read.rs]: Thank you @ArnaudOggy

**Pull request merged**